code/script.js

```diff
@@ -303,41 +351,83 @@ const filterCharacters = (keep) => {
         `No, the person doesn't wear ${attribute}! Remove all that wears ${attribute}`
       )
     }
-  } else if (group === 'other') {
-    // Similar to the one above
-  } else {
+  } else if (category === 'other') {
     if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all persons with yellow hair"
+      alert(
+        `Yes, the person has ${attribute}! Keep all that has ${attribute} `
+        );
     } else {
-      // alert popup that says something like: "NO, the person doesnt have yellow hair! Remove all persons with yellow hair"
+      alert(
+        `No the person don't have ${attribute}. Remove all persons with ${attribute}`
```

Comment:
Careful of spelling/grammar, in this case it would be "doesn't" instead of don't.  Not a big deal just mentioning because it's repeated.